docs/internal/zenhub.md

```diff
@@ -0,0 +1,81 @@
+# ZenHub workflow guidelines
+
+*NOTE*: this document is for internal usage (within Dusk Network) only, and describes the workflow of an internal tool we use for issue management, ZenHub.
+
+This document should bring you up to speed on how to properly use ZenHub, so that the board is always in the right shape and your teammates can easily tell how your work is progressing.
+
+### Table of Contents
+
+* [Pipelines](#pipelines)
+  + [New Issues](#new-issues)
+  + [Icebox](#icebox)
+  + [Backlog](#backlog)
+  + [To Do](#to-do)
+  + [In Progress](#in-progress)
+  + [Review/QA](#review-qa)
+* [Workflow](#workflow)
+  + [Opening and managing Epics](#opening-and-managing-epics)
+  + [Opening and managing issues](#opening-and-managing-issues)
+    - [Blockers](#blockers)
+    - [Pull requests](#pull-requests)
+
+## Pipelines
+
+This section describes the purpose of all the different pipelines found on the `Node-UX` board.
+
+### New Issues
+
+This is where issues will land once they are created, and aren't assigned any pipeline to begin with. Ideally, you shouldn't leave any issues here, as this pipeline is purely just there to catch new issues.
+
+### Icebox
+
+When you are incapable of working on/finishing an issue, they belong in the icebox, which often happens when issues are blocked, either by internal or external factors. Issues coming out of the Icebox generally go into `To Do` once the blocker is cleared.
+
+### Backlog
+
+Contains the issues that can be worked on at any time, but are lower priority. Tickets coming from `New Issues` generally end up here.
+
+### To Do
+
+Issues in this category are ready to be picked up, and can be worked on at any time. Usually contains higher priority issues, or issues associated with an Epic which is in progress.
+
+### In Progress
+
+When an issue is actively being worked on, it belongs here. As a rule of thumb, try to only have a singular issue in progress at all times. If issues are related to one another, it would of course make sense to put them both into In Progress. A good way to tell if this is appropriate, is if they will be fixed in the same PR.
```

Comment:
If issues are related, the developer needs to establish dependencies. This way it is immediately clear what belongs to what. If the issues relate to the same PR, the one associated with such PR is the lowest priority one, so to make sure all the other issues are cleared first